test/types.test.ts

```diff
@@ -101,34 +104,11 @@ Deno.test({
         const file = new InputFile(blob);
         const data = await file.toRaw();
         if (data instanceof Uint8Array) throw new Error("no itr");
-        const values = await readAll(readerFromIterable(data));
+        const values = await convertToUint8Array(data);
         assertEquals(values, new Uint8Array([65, 66, 67, 68])); // ABCD
     },
 });
 
-Deno.test({
-    name: "convert Deno.FsFile to raw",
-    async fn() {
-        let count = 0;
-        const fsfile = new Deno.FsFile(42);
-        const source = stub(fsfile, "read", (buf) => {
-            if (count !== 0) return Promise.resolve(null);
-            let char = 65;
-            for (let i = 0; i < buf.byteLength; i++) buf[i] = char++;
-            count = buf.byteLength;
-            return Promise.resolve(buf.byteLength);
-        });
-        const file = new InputFile(fsfile);
-        const data = await file.toRaw();
-        if (data instanceof Uint8Array) throw new Error("no itr");
-        const values = await readAll(readerFromIterable(data));
-        const expected = new Uint8Array(count);
-        for (let i = 0; i < count; i++) expected[i] = i + 65;
-        assertEquals(values, expected);
-        source.restore();
-    },
-});
-
```

Comment:
Why did you remove this test? I understand that the same code may cover this use case, but it would still be nice to be sure that `Deno.FsFile` is supported, even if it's just a check on the type level and the implementation is redundant.

Reply:
But the API has been deprecated in Deno in favor of using `Deno.open`.

Reply:
Fair enough, so we'd have to find a new way to stub it. I don't currently see how to do that, though. I guess I'll just have to accept this one.